app/mailers/password_reset_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class PasswordResetMailer < ActionMailer::Base
+  # Input:
+  #   user: chef object returned by server
+  def password_reset(user)
+    @user = user
```

Comment:
Yes, by setting them to instance vars, they are available for use in the view template.